brainiak/io.py

```diff
@@ -59,6 +62,10 @@ def load_images_from_dir(in_dir: Union[str, Path], suffix: str = "nii.gz",
     files = sorted(in_dir.glob("*" + suffix))
     for f in files:
         yield nib.load(str(f))
+        logger.info(
+            'file %s is read',
+            f
+        )
```

Comment:
Why not? I think within the io methods are the best places to go.
BTW, I am feeling that `io.load_images_from_dir` and `io.load_images` should be merged.